stable/atlantis/README.md

```diff
@@ -35,6 +35,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `environment`                               | Map of environment variables for the container.                                                                                                                                                                                                                                                           | `{}`    |
 | `imagePullSecrets`                          | List of secrets for pulling images from private registries.                                                                                                                                                                                                                                               | `[]`    |
 | `gitconfig`                                 | Contents of a file to be mounted to `~/.gitconfig`.  Use to allow redirection for Terraform modules in private git repositories.                                                                                                                                                                          | n/a     |
+| `command`                                   | Optionally override the arguments passed to the Atlantis Docker image. By default no arguments are passed which is the same as passing `["server"]`. Must be an array.                                                                                                                                  | n/a     |
```

Comment:
```suggestion
| `command`                                   | Optionally override the [`command` field](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#container-v1-core) of the Atlantis Docker container. If not set, the default Atlantis `ENTRYPOINT` is used. Must be an array.                                                                                                                                  | n/a     |
```

Okay sorry about all the (incorrect) suggestions. This is the final one I swear!

Reply:
No problem. Thank you for review. Awesome feature github suggestion)